tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize.py

```diff
@@ -123,6 +123,8 @@ def compute_stats_layouts(config, model, qscheme=None):
     else:
         fq_configuration = get_configurations_by_qscheme(fq_configuration, qscheme)
 
+    change_configurations_by_model_type(model, config, fq_configuration, hardware_config)
```

Comment:
Can we make it HW-specific, like we will call it for CPU and GPU only while keeping it as is for the rest of the targets (VPU, GNA).